UselessMine.cpp

```diff
@@ -323,6 +329,27 @@ void UselessMine::Event (bz_EventData *eventData)
                 else if (mine.defused)
                 {
                     dieData->killerID = mine.defuserID;
+                    if (playerID != mine.owner)
+                    {
+                        if (!defusalMessages.empty())
+                        {
+                            // The random number used to fetch a random taunting defusal message
+                            int randomNumber = rand() % defusalMessages.size();
+
+                            // Get a random defusal message
+                            std::string defusalMessage = defusalMessages.at(randomNumber);
+                            bz_sendTextMessage(BZ_SERVER, BZ_ALLUSERS, formatDefusalMessage(defusalMessage, bz_getPlayerCallsign(mine.defuserID), owner).c_str());
+                        }
+                        else
+                        {
+                            //let the BD player know that they killed the owner
```

Comment:
Also, could you please add a space between the `//` and the `let`;  that's just my OCD